misc-utils/uuidd.conf.in

```diff
@@ -0,0 +1,6 @@
+# This file is part of uuidd.
+#
+# See tmpfiles.d(5) for details
+#
+d /run/uuidd 2775 uuidd uuidd -
+d /var/lib/libuuid 0755 uuidd uuidd -
```

Comment:
You can also use @runstatedir@ and @localstatedir@ in this template.

Reply:
Ah, good point... Updating in a moment.

Reply:
Well, that fails. We do not have `@localstatedir@` with `meson`, no? The result of `@runstatedir@` is as expected, though.

Reply:
Ah, now I see your comment about @localstatedir@. Is there anything else used in meson?

Reply:
I'll try to fix it.